packages/fxa-customs-server/lib/dataflow.js

```diff
@@ -98,14 +98,31 @@ module.exports = (config, log, fetchRecords, setRecords) => {
         reportOnly,
       });
     } catch (error) {
+      let op;
+
+      if (isFresh(message.publishTime)) {
+        op = 'dataflow.message.error';
+
+        // Call `message.nack` to put the message back on the queue in case
+        // it's a transient error. If the error persists, eventually the
+        // subsequent condition will be entered instead.
+        message.nack();
+      } else {
+        op = 'dataflow.message.error-old';
+
+        // The queue previously carried messages in a different format.
```

Comment:
What's the rollout plan, are we going to set `ignoreOlderThan` to some low number to begin with and gradually increase it as messages in the old format don't overwhelm the queue? Any idea on how many of these messages we could expect?